doc/howto/instances_manage.md

```diff
@@ -76,3 +76,25 @@ There are two ways to prevent accidental deletion of instances:
 
 - To protect a specific instance from being deleted, set [`security.protection.delete`](instance-options-security) to `true` for the instance.
   See {ref}`instances-configure` for instructions.
+
+## Rebuild an instance
+
+If you want to keep the same instance configuration, but you want to freshly start with a new instance filesystem, you have the option to rebuild it.
+
+Before rebuilding an instance, you need to stop it. Also, you can only rebuild instances that have no snapshots. The reason is that rebuilding all the snapshots will require an advanced storage driver logic per backend (e.g, ZFS doesn’t allow us to delete a dataset when there are snapshots associated to it because they are inextricably linked with the original dataset (because of the CoW nature of these snapshots)) that we chose to not implement.
+
+Once these conditions are fulfilled, you can proceed with the rebuild operation. It can be done in three different ways:
+
+- Rebuild the instance with its original image (the one used to create the instance).
```

Comment:
```suggestion
- Rebuild the instance with its original image (the one that was used to create the instance):
```

Reply:
The exact same image? Or might it be a newer version of the image?

Reply:
the exact same one (it uses the fingerprint)